### ISSUE TITLE
Reload certificates every day

### ISSUE THREAD

**Issue by iaguis** [giantswarm/aws-operator#80]
After #79 (upload certificates to S3) we need to re-fetch them every day to allow certificate rolling.

For now, we can use the same [idea of timers](https://github.com/giantswarm/k8s-vm/blob/66665078a8fd07c111789dc9da640dbc89a4f580/worker-cloudconfig.yaml#L230) that's used now in k8s-vm to restart services in machines.

**Comment by xh3b4sd** [comment]
Closing in favour of https://github.com/giantswarm/giantswarm/pull/1755.
